renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
```

Comment:
Add a comment with link to what `config:base` includes?
I guess just github actions?